src/restangular_object_cache.coffee

```diff
@@ -100,7 +100,7 @@ mod.service 'RestangularObjectCache', (Restangular) ->
         _(@objects).chain().values().where(whereClause).value()
 
   wireModel = (relationshipsDefiner, model) ->
-    for definition in relationshipsDefiner.definitions
+    _(relationshipsDefiner.definitions).map (definition) ->
```

Comment:
Turns out the closure for defining the methods wasn't working correctly Using _.map will cause the definition object not to be destroyed and define the method correctly.  Basically, when using "for" the definition object was in the closure, so only it's last value would end up in the defined function for ALL the functions made.